python/clean_content.py

```diff
@@ -53,6 +57,9 @@
 # Read in raw content file
 logger.info('Importing data from %s.', CONTENT_INPUT_PATH)
 
+# Iterate through generator of each content item
+for content_item in islice(items_from_content_file(), 10):
+
```

Comment:
I don't think this is syntactically valid on its own, as the `if` doesn't have anything in it. I think it would be worth combining some later commits in to it.